TAPPC194 in overview, informing users the tap version and eks version (#77)
vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform: docs/deployment_guide/partner_editable/overview.adoc

> This Partner Solution deploys {partner-product-name} ({partner-product-acronym}) {partner-product-version} to the AWS Cloud. This Partner Solution is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}^] on Amazon Elastic Kubernetes Service (Amazon EKS) {eks-version} using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters. For more information about {partner-product-short-name}, refer to https://tanzu.vmware.com/application-platform[VMware Tanzu Application Platform^].

Reviewer: ```suggestion
This Partner Solution deploys {partner-product-name} ({partner-product-acronym} {partner-product-version}) to the AWS Cloud. This Partner Solution is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}^] on Amazon Elastic Kubernetes Service (Amazon EKS {eks-version}) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters. For more information about {partner-product-short-name}, refer to https://tanzu.vmware.com/application-platform[VMware Tanzu Application Platform^].
```

> This Partner Solution deploys {partner-product-name} ({partner-product-acronym}) {partner-product-version} to the AWS Cloud. This Partner Solution is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}^] on Amazon Elastic Kubernetes Service (Amazon EKS {eks-version}) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters. For more information about {partner-product-short-name}, refer to https://tanzu.vmware.com/application-platform[VMware Tanzu Application Platform^].

Reviewer: ```suggestion
This Partner Solution deploys {partner-product-name} ({partner-product-acronym} {partner-product-version}) to the AWS Cloud. This Partner Solution is for users who want a repeatable and customizable reference deployment for https://tanzu.vmware.com/application-platform[{partner-product-name}^] on Amazon Elastic Kubernetes Service (Amazon EKS {eks-version}) using AWS CloudFormation. This guide provides instructions for deploying the {partner-product-short-name} to Amazon EKS clusters. For more information about {partner-product-short-name}, refer to https://tanzu.vmware.com/application-platform[VMware Tanzu Application Platform^].
```

> WARNING: Due to breaking changes from upstream https://github.com/aws-quickstart/quickstart-amazon-eks/releases/tag/v5.0.0[EKS QuickStart CloudFormation Stack^], if you have previously installed TAP through AWS QuickStart or have installed other resources using EKS QuickStart version earlier than 5.0.0. then you need to delete the following CloudFormation Stacks: 

Reviewer: ```suggestion
WARNING: Due to breaking changes from upstream https://github.com/aws-quickstart/quickstart-amazon-eks/releases/tag/v5.0.0[EKS QuickStart CloudFormation Stack^], if you have previously installed {partner-product-acronym} through AWS QuickStart or have installed other resources using EKS QuickStart version earlier than 5.0.0. then you need to delete the following CloudFormation Stacks: 
```
